src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,53 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Represents the response json with the aggregated buildbot data.
+public class BuildInfo {
+
+  // git commit hash of the tested changes commit
```

Comment:
These comments are great! But you should still use Javadoc to document the associated getter methods. Clients of the class don't have direct access to the private fields and documentation on the accessors is directly available in code completion for example.
In order to not repeat too much, I'd suggest having the comments on the getters be the main source of documentation of these fields (with the most details).